addons/folo/common/src/main/java/org/commonjava/indy/folo/conf/FoloConfig.java

```diff
@@ -61,13 +65,44 @@ public Boolean getEnabled()
     {
         return enabled;
     }
-
     @ConfigName( "enabled")
     public void setEnabled( final boolean enabled )
     {
         this.enabled = enabled;
     }
 
+    public boolean isAdminEnabled()
+    {
+        return adminEnabled == null ? DEFAULT_DISABLED : adminEnabled;
```

Comment:
We should set default as true(DEFAULT_ENABLED) here. We can set it false implicitly in  tracking-service mode.

Reply:
updated